docs/azure/sdk/unit-testing-mocking.md

```diff
@@ -231,6 +231,50 @@ var finder = new AboutToExpireSecretFinder(TimeSpan.FromDays(2), secretClient);
 
 This approach is useful when you would like to consolidate the dependency creation and share the client between multiple consuming classes.
 
+## Explore the extension methods in Azure Resource Manager SDKs
+
+For fundamental concepts of Azure Resource Manager SDKs, see [Key Concepts of Azure Resource Manager SDKs](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/resourcemanager/Azure.ResourceManager#key-concepts).
```

Comment:
I don't understand how this relates to unit testing.  Our goal is not to introduce the management libraries - I think this should be removed.

Also - same note on the use of "SDKs"

Reply:
My intention here is to add a brief introduction of the key concepts so that people knows why we have to use extension methods.
I will remove this part.